ui/src/search/queryChange.js

```diff
@@ -2,7 +2,15 @@ import { push, replace } from 'connected-react-router';
 
 import { fetchSearchResults, fetchSearchAggregations } from '../actions/search';
 import { fetchCitationSummary } from '../actions/citations';
-import { isCitationSummaryEnabled } from '../literature/containers/CitationSummarySwitchContainer';
+import { WITH_CITATION_SUMMARY } from '../literature/constants';
+import { CITATION_SUMMARY_ENABLING_PREFERENCE } from '../reducers/user';
+
+function shouldDispatchCitationSummary(state) {
+  return (
+    state.router.location.hash === WITH_CITATION_SUMMARY ||
+    state.user.getIn(['preferences', CITATION_SUMMARY_ENABLING_PREFERENCE])
```

Comment:
```suggestion
    isCitationSummaryEnabled(state) ||
    state.user.getIn(['preferences', CITATION_SUMMARY_ENABLING_PREFERENCE])
```
To have it only in one location